apps/heft/src/cli/test/__snapshots__/CommandLineHelp.test.ts.snap

```diff
@@ -67,7 +67,7 @@ Optional arguments:
 exports[`CommandLineHelp prints the help for each action: start 1`] = `
 "usage: heft start [-h] [-v] [--production] [--locale LOCALE] [-l]
                   [--typescript-max-write-parallelism PARALLEILSM]
-                  [--max-old-space-size SIZE] [--clean] [--storybook]
```

Comment:
Will this info show up when using the storybook plugin? I think it should given how things are formatted, but it's not quite clear.

Reply:
I verified that this shows up in the storybook-react-tutorial sample when running `heft start -h`